pkg/tnf/interactive/oc.go

```diff
@@ -109,3 +111,13 @@ func (o *Oc) GetOptions() []Option {
 func (o *Oc) GetErrorChannel() <-chan error {
 	return o.errorChannel
 }
+
+// GetDoneChannel returns the receive only done channel
+func (o *Oc) GetDoneChannel() <-chan bool {
```

Comment:
This function is not needed.. 
The moment you receive something in the channel, you can return in getOcSession function (line 92)
```
case <- oc :
break
```

Reply:
huh? oc is not a channel.

Reply:
my bad. I thought you're getting the status of the channel and not the channel itself. And I was suggesting to get test directly against the channel. anyway, ignore this comment.